src/current/v22.2/aws-dms.md

```diff
@@ -17,7 +17,7 @@ Using CockroachDB as a source database within AWS DMS is unsupported.
 
 ## Before you begin
 
-Complete the following items before starting this tutorial:
+Complete the following items before starting the DMS migration:
```

Comment:
nit: Maybe a bit redundant to say "the DMS Migration". Wondering if "before starting/running DMS" is better as DMS already stands for database migration service.

Reply:
Thanks! I think I'll keep this one, as it's already used throughout the doc and I think is a nice shorthand for "migration that uses DMS". Open to discussion about it later, though.